msmarco-passage-ranking/files.txt

```diff
@@ -1,2 +1,2 @@
-documents-1k.json.bz2
-documents.json.bz2
\ No newline at end of file
+documents2-1k.json.bz2
```

Comment:
Nitpick: `documents2` -> `documents_with_ids`? No strong feelings about it though.

Reply:
It's the version of the file  ;) I'll remove the previous one after the merge.